.github/workflows/release_production.yml

```diff
@@ -19,6 +19,10 @@ jobs:
       uses: "./.github/actions/setup_node"
     - name: Install Dependencies
       uses: "./.github/actions/npm_install_from_cache"
+    - name: Download Remote Config Backup File
+      uses: ./.github/actions/download_remote_config_backup
+      with:
+        REMOTE_CONFIG_URL: ${{ env.REMOTE_CONFIG_URL }}
```

Comment:
Making a PR is release_production workflow may not be a correct way

Reply:
Can we have it as a separate workflow? May be in a cronjob?

Reply:
the reason that i didn't implement like that is I wanted to download the latest file on release to each environment such as uat, test or staging.
but just on test which is merge to master i'm creating the pr

Reply:
ah okay then